moz/cyclone-flood-zones/flood-extension-maganja-da-costa.json

```diff
@@ -0,0 +1,37 @@
+{
+    "label": "Extensão de inundação (Maganja da Costa) - 16-02-2022",
+    "source": {
+        "type": "geojson",
+        "data": "https://raw.githubusercontent.com/onaio/ingc-mozambique-data/master/moz/cyclone-flood-zones/[Maganja da Costa].geojson",
```

Comment:
@fmusoni The json seems fine by me. The missing bit is the geojson file that will power this layer. You'll need to create a new geojson file inside cyclone-flood-zones name it maganja-da-costa.geojson then paste the contents inside Maganja da Costa.json in it. The last bit should be updating the data property here from `https://raw.githubusercontent.com/onaio/ingc-mozambique-data/master/moz/cyclone-flood-zones/[Maganja da Costa].geojson`
to 
`
https://raw.githubusercontent.com/onaio/ingc-mozambique-data/master/moz/cyclone-flood-zones/maganja-da-costa.geojson 
`
as pointed out on the third point [here](https://github.com/onaio/ingc-mozambique-data/pull/161#issuecomment-1078398135)

Reply:
Thank you for the feedback. For the comment: "You'll need to create a new geojson file inside cyclone-flood-zones name it maganja-da-costa.geojson then paste the contents inside Maganja da Costa.json in it. "

Do you mean create a new geojson file or upload the new geojson file in the folder?
Paste the contents from the geojson file into the json file? This seems, to be, contracdicting Bella's guidance. 

Thank you once again.